### PR TITLE
remove unused variables

### PR DESCRIPTION
**Pull request by denar90**
Should `AuditResult` be removed from types as well?

### CODE REVIEW

**Comment by ebidel** [comment]
Nice. Thanks for the cleanup.

Might be worth adding these to our tsconfig.json so the compile catches it in the future:

```
"noUnusedLocals": true,
"noUnusedParameters": true
```

Looks like there are a couple of other places that get flagged.

File: lighthouse-cli/ask.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@
 const readline = require('readline');
 
 function ask(question: string, options: string[]): Promise<string> {
-  return new Promise((resolve, reject) => {
+  return new Promise((resolve) => {
```
Comment (ebidel) [inline lighthouse-cli/ask.ts]: to be consistent with the rest of the codebase, can you remove the brackets for the single param?

File: lighthouse-cli/chrome-launcher.ts [hunk 2]
```diff
@@ -142,7 +142,7 @@ export class ChromeLauncher {
   }
 
   spawn(execPath: string): Promise<any[]> {
-    return new Promise((resolve, reject) => {
+    return new Promise((resolve) => {
```
Comment (ebidel) [inline lighthouse-cli/chrome-launcher.ts]: remove `()`

File: lighthouse-cli/printer.ts [hunk 3]
```diff
@@ -68,7 +68,7 @@ function createOutput(results: Results, outputMode: OutputMode): string {
  * Writes the output to stdout.
  */
 function writeToStdout(output: string): Promise<{}> {
-  return new Promise((resolve, reject) => {
+  return new Promise(resolve => {
```
Comment (ebidel) [inline lighthouse-cli/printer.ts]: :)

File: lighthouse-cli/ask.ts (hunk 1 above)
Comment (denar90) [inline lighthouse-cli/ask.ts]: oh, sorry, missed these ...

**Pull request merged**